package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -774,4 +872,77 @@ def save_table(self, filename="wbridge_table.pickle"):
            import cPickle
            table = cPickle.load(open(filename))
         """
-        HydrogenBondAnalysis.save_table(self, filename)
+        super().save_table(filename)
+
+    def count_by_type(self):
+        """Counts the frequency of water bridge of a specific type.
+
+        if one atom *A* from *selection 1* is linked to atom *B* from
+        *selection 2* through a bridging water, an entity will be created and
+        the proportion of time that this linkage exits in the whole simulation
+        will be calculated.
+
+        Returns a :class:`numpy.recarray` containing atom indices for *A* and
+        *B*, residue names, residue numbers, atom names (for both A and B) and
+        the fraction of the total time during which the water bridge was
+        detected.
+
+        Returns
+        -------
+        counts : numpy.recarray
+             Each row of the array contains data to define a unique water
+             bridge together with the frequency (fraction of the total time)
+             that it has been observed.
+        """
+        if not self._has_timeseries():
```

Comment:
Please document that the method returns `None` if no timeseries data are available — the user has to `run()` it first.

(I don't like this approach, I'd rather raise a `NoDataError` but to keep it consistent with HydrogenBondAnalysis I'd keep it the way you have.)